proxy/proxy.go

```diff
@@ -1033,11 +1011,17 @@ func (p *Proxy) do(ctx *context) (err error) {
 				p.log.Errorf("stacktrace of panic caused by: %v:\n%s", r, stack())
 			})
 
-			err = fmt.Errorf("panic caused by: %v", r)
+			perr := &proxyError{
+				err: fmt.Errorf("panic caused by: %v", r),
+			}
+			p.makeErrorResponse(ctx, perr)
+			err = perr
 		}
 	}()
 
 	if ctx.executionCounter > p.maxLoops {
+		// TODO(sszuecs): think about setting status code to 465 (AWS redirect loop) or similar
```

Comment:
I think it is 463, lets maybe add a link https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-troubleshooting.html#http-463-issues

Reply:
yes maybe that. I remember 465, but there is only ERR_TOO_MANY_REDIRECTS and nothing about the status code.
It could be that it always adds to XFF and entry, which would be 463.
https://aws.amazon.com/premiumsupport/knowledge-center/redirect-http-https-elb/ has no status code

Reply:
With this weak data I will not add wrong docs to it.